src/custom_typings/hubot-slack.d.ts

```diff
@@ -50,20 +66,28 @@ declare module "hubot-slack" {
     icon_emoji?: string;
   }
 
-  interface SlackBot extends Adapter {
-    client: {
-      rtm: {
-        dataStore: MemoryDataStore;
-      }
+  interface SlackBotClient {
+    rtm: {

```

Comment:
Indeed it is! I just need to add the definitions for the `@slack/client` `RtmClient`.